test/integration/content-sidebar/TaskForm.e2e.test.js

```diff
@@ -48,19 +49,8 @@ describe('Create Task', () => {
                 getSubmitButton().click();
             });
 
-            // modal should close
-            cy.getByTestId('create-task-modal').should('not.exist');
-
-            // validate task appears in feed
-            // note that in the test environment task create fails with default token
-            // but the card temporarily appears
-            cy.getByTestId('activityfeed').within(() => {
-                cy.getByTestId('task-card')
-                    .last()
-                    .within(() => {
-                        cy.contains('valid e2e task').should('exist');
-                    });
-            });
+            // submit button should be in loading state
+            getSubmitButton().should('have.class', 'is-loading');
```

Comment:
Can we add an e2e test for the loading state as the deleted portion of the test did? can rename the test flow to account for the fact that we aren't going to get a write token for e2e tests any time soon: `it(shows error state after receiving server error)`

Reply:
Since the loading state happens in the modal now to account for showing an error in the modal, I took out the pending task in the activity feed, so that deleted part should stay deleted. I think we can still test the task showing up in the feed if we stub out the creation response to return some sample task data and verify that the sample task shows up in the feed.

Reply:
That would be good, but we can also test for the error showing up right now because the server is going to error with a read token.